docs/release-notes/NuGet-5.5.md

```diff
@@ -0,0 +1,106 @@
+---
+title: NuGet 5.5 Release Notes
+description: Release notes for NuGet 5.5 including new features, bug fixes, and DCRs.
+author: karann-msft
+ms.author: karann
+ms.date: 03/19/2020
+ms.topic: conceptual
+---
+
+# NuGet 5.5 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version| Available in .NET SDK(s)|
+|:---|:---|:---|
+| [**5.5.0**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.5](https://visualstudio.microsoft.com/downloads/) | [3.1.200](https://dotnet.microsoft.com/download/dotnet-core/3.1)<sup>1</sup> |
+
+<sup>1</sup>Installed with Visual Studio 2019 with .NET Core workload
+
+## Summary: What's New in 5.5
```

Comment:
* the points we highlight in the summary are the list of most impactful changes we made in the release. We also typically call out the value prop to articulate said impact
* Is this ordered by impact/importance? 
* the associated issue should be linked - [see 5.4 release notes](https://docs.microsoft.com/en-us/nuget/release-notes/nuget-5.4)

Reply:
> * the points we highlight in the summary are the list of most impactful changes we made in the release. We also typically call out the value prop to articulate said impact
> * Is this ordered by impact/importance?

Yes, the issues mentioned under `Summary` are ordered by impact to the best of my knowledge and feedback from team members. I really appreciate additional feedback to make the release notes even better.

> * the associated issue should be linked - [see 5.4 release notes](https://docs.microsoft.com/en-us/nuget/release-notes/nuget-5.4)

done. added a link to [ZenHub release](https://app.zenhub.com/workspaces/nuget-client-team-55aec9a240305cf007585881/reports/release?release=5e0e5fbd021f7aa0ec95db18) at the end of the current release notes.